go.mod

```diff
@@ -81,6 +81,7 @@ require (
 	github.com/rs/cors v1.11.0
 	github.com/secure-io/sio-go v0.3.1
 	github.com/shirou/gopsutil/v3 v3.24.5
+	github.com/stretchr/testify v1.9.0
```

Comment:
don't add testify this is a terrible dependency, just use Go's testing package.

Reply:
okay